CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.15...3.26)
+project(${SKBUILD_PROJECT_NAME} LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 17 CACHE STRING "C++ version selection")
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Force macOS toolchains to use C++17
+set(CMAKE_OSX_DEPLOYMENT_TARGET 10.15)
+
+set(PYBIND11_FINDPYTHON ON)
+find_package(pybind11 CONFIG REQUIRED)
+
+set(Boost_DEBUG ON)
+find_package(Boost 1.53 REQUIRED)
```

Comment:
Do you know what Boost support cycles are or does it not matter what the min version of boost is as long as it works?

Reply:
The latter. This package only uses the odeint part of boost which is header only and this is the first version that included odeint.

Reply:
Makes sense, not sure if that is noted in any documentation or if it's even important enough to be noted anywhere.

Reply:
That's a good point. I should add a note about the minimum version required in the contributing docs.

Reply:
Ah I already had a note about the minimum required version in the contributing docs.